stix_shifter_modules/nozomi/stix_translation/transformers.py

```diff
@@ -0,0 +1,135 @@
+import re
+from stix_shifter_utils.utils import logger
+from datetime import datetime, timezone
+from stix_shifter_utils.stix_translation.src.utils.transformers import ValueTransformer
+
+LOGGER = logger.set_logger(__name__)
+connector = __name__.split('.')[1]
+
+
+class ChainNameValue(ValueTransformer):
+    """A value transformer to add default 'kill_chain_name' as 'mitre-attack' with obj
+    Example:
+        Input: 'Command and Control'
+        Output: [{'kill_chain_name': 'mitre-attack', 'phase_name': 'Command and Control'}]
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if obj:
+                return [{"kill_chain_name": "mitre-attack", "phase_name": obj}]
+        except ValueError:
+            LOGGER.error("%s connector error, cannot return obj value %s with default param 'kill_chain_name'",
+                         connector, obj)
+        return None
+
+
+class ValidateMacAddr(ValueTransformer):
+    """Regex to check valid MAC address
+    Example:
+        Input: Invalid mac address
+        Output: None
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            pattern = re.compile(r'^(([0-9a-fA-F]{2}[:-]){5}([0-9a-fA-F]{2})|([0-9a-fA-F]{3}[\.]){3}([0-9a-fA-F]{3}))$')
+            if re.search(pattern, str(obj)):
+                return obj
+            return None
+        except ValueError:
+            LOGGER.error("%s connector error, cannot validate mac address: %s", connector, obj)
+            raise
+
+
+class ToFindingType(ValueTransformer):
+    """A value transformer for expected finding_type value
+    Example:
+        Input: empty string or other than empty string
+        Output: alert or threat
+    """
+    @staticmethod
+    def transform(obj):
+        try:
+            if obj in ("", "alert"):
+                return "alert"
+            return "threat"
+        except ValueError:
+            LOGGER.error("%s connector error, cannot convert input to finding type : %s", connector, obj)
+            raise
+
+
+class ToSeverityValue(ValueTransformer):
+    """A value transformer for converting severity value
+    Example:
+        Input: 9.0 or 9 or '9.0'
```

Comment:
If this transformer is only being used on one field (risk it looks like) why do we need three different formats?

Reply:
We are expecting  int, float and string value for this in API output

Reply:
Can you provide an example response for each type? I don't doubt it's possible, but if it is, I'd like to have some documentation to go back to and say "This is why it was done this way".

Reply:
I've looked through the examples sent via mail. I'm wondering if you could create a unit test for each type.